transforms/language/doc_quality/python/pyproject.toml

```diff
@@ -0,0 +1,46 @@
+[project]
+name = "dpk_doc_quality_transform_python"
+version = "0.2.1.dev0"
+requires-python = ">=3.10"
+description = "Document Quality Python Transform"
+license = {text = "Apache-2.0"}
+readme = {file = "README.md", content-type = "text/markdown"}
+authors = [
+    { name = "Daiki Tsuzuku", email = "dtsuzuku@jp.ibm.com" }
+]
+dependencies = [
+    "data-prep-toolkit==0.2.1.dev0",
+    "sentencepiece==0.2.0",
+    "kenlm @ https://github.com/kpu/kenlm/archive/master.zip",
```

Comment:
so its not the model, but the code that is LGPL. Need to discuss

Reply:
done